internal/adapter/zep/client.go

```diff
@@ -54,7 +54,7 @@ func (zep *CalendarAPI) Name() string {
 	return "ZEP CalDav API"
 }
 
-func (zep *CalendarAPI) Initialize(ctx context.Context, calendarID string, config map[string]interface{}) error {
```

Comment:
It would really be useful if an adapter implementation could import the `adapter` package, then would could simply assert that the adapter implements the Configurable interface using `var _ adapter.Configurable = &CalendarAPI{}`. But unfortunately that ends with an import cycle.

I'll try to come up with a solution later on -> separate PR.